tests/unittests/benchmark/task/test_task_wrapper.py

```diff
@@ -0,0 +1,87 @@
+from typing import Any, Dict, List
+
+import pytest
+from orion.benchmark.task.base import BaseTask
+from orion.benchmark.task.task_wrapper import FixTaskDimensionsWrapper, TaskWrapper
+from orion.core.io.space_builder import SpaceBuilder
+
+
+class DumbTask(BaseTask):
+    def __init__(self, max_trials: int, seed: int = None, a: int = 100, b: int = 10, c: int = 1):
+        super().__init__(max_trials=max_trials)
+        self.a = a
+        self.b = b
+        self.c = c
+        self.seed = seed
+
+    def call(self, x: Dict) -> List[Dict]:
+        y = self.a * x["a"] + self.b * x["b"] + self.c * x["c"]
+        return [dict(name="dumb_task", type="objective", value=y)]
+
+    def get_search_space(self) -> Dict[str, str]:
+        return {
+            "a": "uniform(0, 10, discrete=True)",
+            "b": "uniform(0, 10, discrete=True)",
+            "c": "uniform(0, 10, discrete=True)",
+        }
+
+    @property
+    def configuration(self) -> Dict[str, Any]:
+        return {
+            type(self).__qualname__: dict(
+                max_trials=self.max_trials, seed=self.seed, a=self.a, b=self.b, c=self.c,
+            )
+        }
+
+
+class TestFixDimensionsWrapper:
+    def test_init(self):
```

Comment:
There should be test_init_with _object and test_init_with_config.

Reply:
You can simply test the attributes at this point and leave the call for another test.

Reply:
Parametrized the test with an additional "task_as_dict" argument which does that. Marking this as resolved.